src/io.jl

```diff
@@ -1,3 +1,5 @@
+using EllipsisNotation
+
```

Comment:
Similar to Trixi.jl, the `using X` statements should all be in the `src/Trixi2Img.jl` file.